main_ISTA.py

```diff
@@ -49,7 +59,10 @@ def __init__(self, data: Dataset, num_subsets: int = 7, step_size: float = 1e-6,
         step_size_rule = ConstantStepSize(step_size)    # ISTA default step_size is 0.99*2.0/F.L
         g = IndicatorBox(lower=1e-6, accelerated=False) # "non-negativity" constraint
```

Comment:
```suggestion
        g = IndicatorBox(lower=0, accelerated=False) # "non-negativity" constraint
```

Reply:
my guess is we can't use zero as the algorithm doesn't handle zero division errors/NaNs?

Reply:
It's an indicator, should just threshold, no division. But I haven't checked.

Reply:
Maybe it was a left over from trying to debug. I tested with `0` and it works just fine.